admin/load.php

```diff
@@ -820,13 +833,17 @@ function perflab_plugin_admin_notices() {
 			esc_attr( $available_module_names[0] )
 		);
 		$message .= ' ';
-		$message .= esc_html__( 'Please click the following button to install and activate the relevant plugin in favor of the module. This will not impact any of the underlying functionality.', 'performance-lab' );
+		$message .= $additional_message;
```

Comment:
Better to do the escaping here, as late as possible:

```suggestion
		$message .= esc_html( $additional_message );
```

The `esc_html__()` instances can be replaced with `__()`.

Reply:
Thanks for quick feedback. Addressed in https://github.com/WordPress/performance/pull/944/commits/e694fc091a243f1235af257700a302a45be2b42c